src/main/rx/crsf.h

```diff
@@ -22,27 +22,37 @@
 #define CRSF_PORT_MODE          MODE_RXTX
 
 #define CRSF_MAX_CHANNEL        16
+#define CRSF_MSP_RX_BUF_SIZE 128
+#define CRSF_MSP_TX_BUF_SIZE 128
 
 typedef enum {
     CRSF_FRAMETYPE_GPS = 0x02,
     CRSF_FRAMETYPE_BATTERY_SENSOR = 0x08,
     CRSF_FRAMETYPE_LINK_STATISTICS = 0x14,
     CRSF_FRAMETYPE_RC_CHANNELS_PACKED = 0x16,
     CRSF_FRAMETYPE_ATTITUDE = 0x1E,
-    CRSF_FRAMETYPE_FLIGHT_MODE = 0x21
-} crsfFrameType_e;
+    CRSF_FRAMETYPE_FLIGHT_MODE = 0x21,
+    CRSF_FRAMETYPE_DEVICE_PING = 0x28,
+    CRSF_FRAMETYPE_DEVICE_INFO = 0x29,
+    CRSF_FRAMETYPE_MSP_REQ = 0x7A,   // response request using msp sequence as command
+    CRSF_FRAMETYPE_MSP_RESP = 0x7B,  // reply with 60 byte chunked binary
+    CRSF_FRAMETYPE_MSP_WRITE = 0x7C  // write with 60 byte chunked binary
+} crsfFrameTypes_e;
```

Comment:
This should remain `crsfFrameType_e` (no `s`).

Reply:
Picked this up and was confused in the merge conflict.  Will correct.

Reply:
Amended